src/js/helpers/test-helper.js

```diff
@@ -0,0 +1,5 @@
+export function eventHandlerTest (wrapper, handlingEvent, eventData) {
```

Comment:
The function name **MUST** start with the verb, except it's not a  boolean one - then you should start it with `is`

Reply:
I'd call it `isEmittedOnce(wrapper, event, payload)`

Reply:
Also why this func is not a member of test helper class?

Reply:
And also why haven't you updated another code that duplicates this logic?

Reply:
ok